pkg/jx/cmd/step_helm_apply.go

```diff
@@ -242,36 +242,47 @@ func (o *StepHelmApplyOptions) Run() error {
 func (o *StepHelmApplyOptions) applyTemplateOverrides(chartName string) error {
 	log.Infof("Applying chart overrides\n")
 	templateOverrides, err := filepath.Glob(chartName + "/../*/templates/*.yaml")
```

Comment:
wonder if that line should be:
```
	templateOverrides, err := filepath.Glob(filepath.Join(chartName, "..", "*", "templates", "*.yaml"))
```

Reply:
possibly, not sure.
in this case it works as it goes into the loop and fails to copy the files so it did match stuff

Reply:
docs for glob says it calls [`filepath.Match`](https://golang.org/pkg/path/filepath/#Match) which has some interesting behaviour on windows.  Also if the chartName contains special characters like `-` then they should be escaped (I think the chartname may contain a dash.

Reply:
`-` only needs escaping in a choice `[...]`